packages/cli/src/Validate.ts

```diff
@@ -33,6 +34,7 @@ ${bold('Usage')}
 
 ${bold('Options')}
 
+    --config   Custom path to your Prisma config file
```

Comment:
Let's add this to other commands as well, see [ORM-607](https://linear.app/prisma-company/issue/ORM-607/when-prismaconfigts-is-detected-disable-automatic-env-file-loading)